tools/scripts/generic.mk

```diff
@@ -37,3 +37,8 @@ include ../../tools/scripts/linux.mk
 endif
 
 endif
+
+# Use " >make list | tr ' ' '\n' " to see the file list with new lines instead
```

Comment:
I would output with newline by default, this way you can easily pass the file list to grep and other tools using a pipe.

Reply:
I tried, but I couldn't split it with new lines

Reply:
i understand bash and makefile syntaxes are ugly but i'm pretty sure there is a way...

Reply:
I didn't find a way do it using makefile syntaxes that works both on windows and linux.
I make it work for linux using bash command.